bin/proving-service/README.md

```diff
@@ -28,48 +28,49 @@ This will spawn a worker using the hosts and ports defined in the command option
 
 ## Proxy
 
-First, you need to create a configuration file for the proxy with:
+To start the proxy service, you will need to run:
 
 ```bash
-miden-proving-service init
+miden-proving-service start-proxy [worker1],[worker2],...,[workerN]
+miden-proving-service start-proxy 0.0.0.0:8084 0.0.0.0:8085
 ```
 
-This will create the `miden-proving-service.toml` file in your current directory. This file will hold the configuration for the proxy. You can modify the configuration by changing the host and ports of the services, the maximum size of the queue, among other options. An example configuration is:
+This command will start the proxy using the workers passed as arguments. The workers should be in the format `host:port`. If no workers are passed, the proxy will start without any workers and will not be able to handle any requests until one is added through the `miden-proving-service add-worker` command.
+
+You can customize the proxy service by setting the following environment variables:
 
-```toml
+```bash
 # Host of the proxy server
-host = "0.0.0.0"
+MPS_HOST="0.0.0.0"
 # Port of the proxy server
-port = 8082
+MPS_PORT="8082"
+# Host of the workers update endpoint
+MPS_WORKERS_UPDATE_PORT="8083"
 # Timeout for a new request to be completed
-timeout_secs = 100
+MPS_TIMEOUT_SECS="100"
 # Timeout for establishing a connection to the worker
-connection_timeout_secs = 10
+MPS_CONNECTION_TIMEOUT_SECS="10"
 # Maximum amount of items that a queue can handle
-max_queue_items = 10
+MPS_MAX_QUEUE_ITEMS="10"
 # Maximum amount of retries that a request can take
-max_retries_per_request = 1
+MPS_MAX_RETRIES_PER_REQUEST="1"
 # Maximum amount of requests that a given IP address can make per second
-max_req_per_sec = 5
+MPS_MAX_REQ_PER_SEC="5"
 # Time to wait before checking the availability of workers
-available_workers_polling_time_ms = 20
+MPS_AVAILABLE_WORKERS_POLLING_TIME_MS="20"
 # Interval to check the health of the workers
-health_check_interval_secs = 1
+MPS_HEALTH_CHECK_INTERVAL_SECS="1"
 # Host of the metrics server
-prometheus_host = "127.0.0.1"
+MPS_PROMETHEUS_HOST="127.0.0.1"
 # Port of the metrics server
-prometheus_port = 6192
-```
-
-Then, to start the proxy service, you will need to run:
-
-```bash
-miden-proving-service start-proxy [worker1] [worker2] ... [workerN]
+MPS_PROMETHEUS_PORT="6192"
+# Log level
+RUST_LOG="info"
 ```
 
-This command will start the proxy using the workers passed as arguments. The workers should be in the format `host:port`. If no workers are passed, the proxy will start without any workers and will not be able to handle any requests until one is added through the `miden-proving-service add-worker` command.
+An example `.env` file is provided in the crate's root directory.
```

Comment:
I would add an example of how `.env` file can be loaded on Linux based system (e.g., via the `source` command).